app/models/community_users.rb

```diff
@@ -0,0 +1,32 @@
+class CommunityUsers
+
+  def initialize(params, users)
+    @users         = users
+    @role          = params[:role].presence
+    @interest      = params[:interest].presence
+    @availability  = params[:availability].presence
+    @search        = params[:search].presence
+    @location      = params[:location].presence
+    @page          = params[:page].presence
+  end
+
+  def all
```

Comment:
Can you come up with tests for this? You can stub out all the class methods on `User`, I think it would just be helpful to ensure the contract, so that if maybe in the future someone removes one of these class methods, this is caught by our tests.